tests/cli/cdk_smoke_tests/cdk-basic.bats

```diff
@@ -60,11 +60,13 @@ setup_file() {
 @test "Pack connector" {
     # Pack when package meta doesn't exist
     cd $CONNECTOR_DIR
+    echo "# My Cool Connector" > README.md
```

Comment:
I think that instead of this we should: either put a readme in connector template and put a readme in `connector/json-test-connector`

Reply:
Well thought!